app/models/hyrax/collection_type.rb

```diff
@@ -82,39 +83,31 @@ def self.any_nestable?
       where(nestable: true).any?
     end
 
-    # Find or create the default collection type as defined by:
+    # Find or create the user collection type as defined by:
     #
     # * USER_COLLECTION_MACHINE_ID
     # * USER_COLLECTION_DEFAULT_TITLE
-    # * Hyrax::CollectionTypes::CreateService::DEFAULT_OPTIONS
+    # * Hyrax::CollectionTypes::CreateService::USER_COLLECTION_OPTIONS
     #
     # @see Hyrax::CollectionTypes::CreateService
     #
     # @return [Hyrax::CollectionType] where machine_id = USER_COLLECTION_MACHINE_ID
-    def self.find_or_create_default_collection_type
-      find_by(machine_id: USER_COLLECTION_MACHINE_ID) || Hyrax::CollectionTypes::CreateService.create_collection_type
+    def self.find_or_create_user_collection_type
+      find_by(machine_id: USER_COLLECTION_MACHINE_ID) || Hyrax::CollectionTypes::CreateService.create_user_collection_type
     end
+    singleton_class.send(:alias_method, :find_or_create_default_collection_type, :find_or_create_user_collection_type)
```

Comment:
Is there some reason we need to have an alias?  Can we pick a single name for the method?

Reply:
In cases where a default is used, it is more meaningful to have a method that implies that a default is being set.